ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/util/DataStructureUtil.java

```diff
@@ -276,13 +278,36 @@ public static Exit randomExit(int seed) {
     return new Exit(randomUnsignedLong(seed), randomUnsignedLong(seed++), BLSSignature.random());
   }
 
+  public static Transfer randomTransfer() {
+    return new Transfer(
+        randomUnsignedLong(),
+        randomUnsignedLong(),
+        randomUnsignedLong(),
+        randomUnsignedLong(),
+        randomUnsignedLong(),
+        BLSPublicKey.randomPublicKey(),
```

Comment:
Once #403 has been merged, you can use `BLSPublicKey.random()`. Meanwhile, the codebase uses `Bytes48.random()` for this, and public keys everywhere are Bytes48 type (#403 fixes this).

Reply:
I think #403 was merged after v0.4-integration was branched off so leaving it as is until the branch is updated.

Reply:
That's ok, but in that case you should use `Bytes48.random()` here rather than `BLSPublicKey` to be consistent with the rest of the codebase, and there oughtn't be any need to modify the BLSPublicKey class.

Reply:
Alternatively, perhaps @schroedingerscode could rebase the 0.4 branch on the latest master, then you can take advantage of the updated BLSPublicKey goodness, and future merges will be cleaner.

Reply:
Yeah, I pinged him on Gitter :-)

Reply:
Updated.